app/components/channel_list/__snapshots__/index.test.tsx.snap

```diff
@@ -70,7 +70,298 @@ exports[`components/channel_list should match snapshot 1`] = `
+            "marginLeft": 5,
+            "marginTop": -2,
+          },
+        ]
+      }
+    />
+  </View>
+  <View
+    onMoveShouldSetResponder={[Function]}
+    onMoveShouldSetResponderCapture={[Function]}
+    onResponderEnd={[Function]}
+    onResponderGrant={[Function]}
+    onResponderMove={[Function]}
+    onResponderReject={[Function]}
+    onResponderRelease={[Function]}
+    onResponderStart={[Function]}
+    onResponderTerminate={[Function]}
+    onResponderTerminationRequest={[Function]}
+    onStartShouldSetResponder={[Function]}
+    onStartShouldSetResponderCapture={[Function]}
+  >
+    <View
+      accessible={true}
+      focusable={true}
+      onClick={[Function]}
+      onResponderGrant={[Function]}
+      onResponderMove={[Function]}
+      onResponderRelease={[Function]}
+      onResponderTerminate={[Function]}
+      onResponderTerminationRequest={[Function]}
+      onStartShouldSetResponder={[Function]}
+    >
+      <View
+        style={
+          Object {
+            "display": "flex",
+            "flexDirection": "row",
+          }
+        }
+      >
+        <Icon
+          name="message-text-outline"
+          style={
+            Object {
+              "color": "#ffffff",
+              "fontSize": 24,
+              "lineHeight": 28,
+            }
+          }
+        />
+        <Text
+          style={
+            Array [
+              Array [
+                Object {
+                  "fontFamily": "OpenSans-SemiBold",
+                  "fontSize": 16,
+                  "fontWeight": "600",
+                  "lineHeight": 24,
+                },
+              ],
+              Object {
+                "color": "#ffffff",
+                "paddingLeft": 12,
+              },
+            ]
+          }
+        >
+          Threads
+        </Text>
+      </View>
+    </View>
+  </View>
+  <View
+    style={
+      Object {
+        "alignItems": "center",
+        "flex": 1,
+        "justifyContent": "center",
+        "padding": 20,
+      }
+    }
+  >
+    <View
+      style={
+        Object {
+          "alignItems": "center",
+          "backgroundColor": "rgba(255,255,255,0.08)",
+          "borderRadius": 60,
+          "height": 120,
+          "justifyContent": "center",
+          "width": 120,
+        }
+      }
+    >
+      <Icon
+        name="alert-circle-outline"
+        style={
+          Object {
+            "color": "rgba(255,255,255,0.48)",
+            "fontSize": 72,
+            "lineHeight": 72,
+          }
+        }
+      />
+    </View>
+    <Text
+      style={
+        Array [
+          Object {
+            "fontFamily": "Metropolis-SemiBold",
+            "fontSize": 20,
+            "fontWeight": "600",
+            "lineHeight": 28,
+          },
+          Object {
+            "color": "#ffffff",
+            "marginTop": 20,
+            "textAlign": "center",
+          },
+        ]
+      }
+    >
+      Couldn't load categories for 
```

Comment:
Nit: Probably it is just missing the team name on the state, but here seems is missing something.